features/zip_files_storage.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Storage of DIP ZIP files
+  The CCA DIP access application will have the option to split the storage of the
+  uploaded DIPs into different locations of the local filesystem. Each location
+  will store a configurable amount of data. This configuration will be made via
+  two environment variables:
+    - STORAGE_LOCATIONS: list of locations paths separated by comma.
+    - STORAGE_LOCATION_SIZE: maximun size of each location.
+
+  Scenario: Default configuration
+    Given an instance running
+    When the STORAGE_LOCATIONS and STORAGE_LOCATION_SIZE variables are not configured in the environment
+    Then the instance will use the "media" folder inside the instance directory to store the ZIP files
+    And the instance won't check the size used by ZIP files on that folder
+
+  Scenario: Custom configuration
+    Given an instance running
+    When the STORAGE_LOCATIONS and/or STORAGE_LOCATION_SIZE variables are configured in the environment
+    Then the instance will use the locations indicated in STORAGE_LOCATIONS to store the ZIP files
+    And the instance will limit each location size to the STORAGE_LOCATION_SIZE env. var. value
+
+  Scenario: Behavior
```

Comment:
I want to insert the two scenarios above into this 1 behaviour scenario but it's awkward. i see why you have done it this way.

Reply:
Yes, I tried, it was really a mess ;)